Frontend/ui-driver/src/Flow.purs

```diff
@@ -289,14 +289,14 @@ enterOTPFlow = do
       modifyScreenState $ EnterOTPScreenType (\enterOTPScreen → enterOTPScreen { data { tokenId = resp_resend.authId, attemptCount = resp_resend.attempts}})
       enterOTPFlow
 
--- TODO :: As per the api response check the floe for driver registration
-registrationStatusFlow :: FlowBT String Unit
-registrationStatusFlow = do
-  let startTime = getTime unit
-  -- resp <- Remote.driverRegistrationStatus (DriverResgistrationStatusReq {})
-  let endTime = getTime unit
-  let registrationStatus = driverRegistrationStatus
-  if registrationStatus then currentRideFlow Nothing else permissionsScreenFlow Nothing Nothing
+-- TODO :: As per the api response check the flow for driver registration, will need in future
```

Comment:
what's the future need of this?

Reply:
just commented code which are defined but not being used